src/relayFeeCalculator/chain-queries/ethereum.ts

```diff
@@ -1,87 +1,12 @@
 import { DEFAULT_LOGGER, Logger } from "../relayFeeCalculator";
 import { providers } from "ethers";
 import QueryBase from "./baseQuery";
-
-// Note: these are the mainnet addresses for these symbols meant to be used for pricing.
-export const SymbolMapping: { [symbol: string]: { address: string; decimals: number } } = {
-  USDC: {
-    address: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
-    decimals: 6,
-  },
-  WETH: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  ETH: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  OETH: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  AETH: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  KOV: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  KOR: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
-  ARETH: {
-    address: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    decimals: 18,
-  },
```

Comment:
Not sure if we really need these definitions? I searched through the FE if these symbols are used but couldn't find any places where we use them, so I opted to not include them in `TOKEN_SYMBOLS_MAP`. Correct me if I am wrong.